package.json

```diff
@@ -31,9 +31,12 @@
     "@angular/compiler-cli": "~15.1.0",
     "@angular/language-service": "~15.1.0",
     "@nx/angular": "17.3.0-beta.8",
+    "@nx/devkit": "^18.0.1",
```

Comment:
Do we need `v18.0.1` here or can we use `"17.3.0-beta.8"` too? 

If yes please update all nx packages to v18.x.x or do in in a seperate PR beforehand.